src/blocks/Tabs/TabsTextContent/TabsTextContent.tsx

```diff
@@ -21,31 +21,29 @@ export const TabsTextContent = ({
     centered,
     contentSize = 's',
     showMedia,
-    data,
+    data: {media, title, text, additionalInfo, link, links, buttons, list},
```

Comment:
but data could be undefined, no?

Reply:
`data` is required prop, inside data we have partial fields like `media` and we check his existence.